std_msgs/Header header
bluerov2_dock/marker_pose[] locations
bool detection_flag
bool controller_param
bool marker_switch